test/integ_tests/qiskit/rig_pulse_device_capabilities.json

```
@@ -8114,7 +8114,7 @@
       "name": "braket.device_schema.pulse.pulse_device_action_properties",
       "version": "1"
     },
-    "nativeGateCalibrationsRef": "https://dsq6c4wdong7p.cloudfront.net/native_gate_calibrations/device:arn:aws:braket:us-west-1::device/qpu/rigetti/Aspen-M-3?versionId=.sbF0vCCQLMmN2bPi_YILDyNv7VUJUut",
+    "nativeGateCalibrationsRef": "https://dsq6c4wdong7p.cloudfront.net/native_gate_calibrations/device:arn:aws:braket:us-west-1::device/qpu/rigetti/Ankaa 2?versionId=.sbF0vCCQLMmN2bPi_YILDyNv7VUJUut",
```

I believe the integ test is failing in CI because this URL has a space in it:
```suggestion
    "nativeGateCalibrationsRef": "https://dsq6c4wdong7p.cloudfront.net/native_gate_calibrations/device:arn:aws:braket:us-west-1::device/qpu/rigetti/Ankaa-2?versionId=.sbF0vCCQLMmN2bPi_YILDyNv7VUJUut",
```
I think that should fix the test - unless we want to redownload this whole .json file so that it matches the Ankaa-2 device.